pkg/agent/testdata/RawUbuntu/line70.sh

```diff
@@ -135,6 +135,7 @@ configureHTTPProxyCA() {
 
 configureCustomCaCertificate() {
     mkdir -p /opt/certs
+    chmod 1755 /opt/certs
```

Comment:
do we put anything else in this dir?

Reply:
As far as I know we only use this for custom ca feature. I checked RP/AB and it only appears in paths related to custom ca.

Reply:
might be worth leaving a comment clarifying that we don't use this dir for anything else

Reply:
Sure, added